src/main/java/core/mate/academy/service/impl/BulldozerProducerImpl.java

```diff
@@ -0,0 +1,36 @@
+package core.mate.academy.service.impl;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducerImpl implements MachineProducer<Bulldozer> {
+    private final List<Bulldozer> bulldozers;
+
+    public BulldozerProducerImpl() {
+        bulldozers = new ArrayList<>();
+        addBulldozers();
+    }
+
+    @Override
+    public List<Bulldozer> get() {
+        return new ArrayList<>(bulldozers);
+    }
+
+    private void addBulldozers() {
```

Comment:
The method `addBulldozers` should not begin with an empty line. Remove the empty line at the start of the method implementation.